lib/cryptoexchange/exchanges/novaexchange/models/market_pair.rb

```diff
@@ -0,0 +1,14 @@
+module Cryptoexchange::Exchanges
+  module Novaexchange
+    module Models
+      class MarketPair < Cryptoexchange::Models::MarketPair
+        attr_accessor :name
+
+        def initialize(params={})
+          super
+          @name = params[:name]
```

Comment:
I think it's better to do it as a `hack` with a `FIXME` and `Issue` on GitHub so that we remember to take it out later